java/vector/src/main/codegen/templates/UnionVector.java

```diff
@@ -649,4 +649,12 @@ public boolean equals(int index, ValueVector to, int toIndex) {
       }
       return leftVector.equals(index, rightVector, toIndex);
     }
+
+    @Override
+    public void copyFrom(int fromIndex, int thisIndex, ValueVector from) {
```

Comment:
there's already an impl of copyFrom in UnionVector, right ?

https://github.com/apache/arrow/blob/master/java/vector/src/main/codegen/templates/UnionVector.java#L367

Reply:
Nice catch. Thank you @pravindra